Clients/src/App.tsx

```diff
@@ -22,7 +22,7 @@ function App() {
       <CssBaseline />
       <Routes>
         <Route path="/" element={<Dashboard />}>
-          <Route path="/" element={<Home />} />
+          <Route path="/" element={<Home projects={projects} />} />
           <Route path="/compliance-tracker" element={<ComplianceTracker complianceMetrics={complianceMetrics} complianceDetails={complianceDetails} />} />
```

Comment:
@melpsh Please apply the same thing as @popovaevgeniya here in your next **pull request**.
We should not pass props from inside the `App.tsx`. At least not for now.